src/engine/client/backend_sdl.cpp

```diff
@@ -198,33 +198,33 @@ bool CCommandProcessorFragment_General::RunCommand(const CCommandBuffer::SComman
 }
 
 // ------------ CCommandProcessorFragment_SDL
-void CCommandProcessorFragment_SDL::Cmd_Init(const SCommand_Init *pCommand)
+void CCommandProcessorFragmentSDL::Cmd_Init(const SCommand_Init *pCommand)
```

Comment:
```suggestion
void CCommandProcessorFragmentSdl::Cmd_Init(const SCommand_Init *pCommand)
```
Also the comment above.